src/field.rs

```diff
@@ -498,6 +500,7 @@ impl<F: Field> Shl<usize> for &Element<F> {
 impl<F: Field> fmt::Display for Element<F> {
     fn fmt(&self, f: &mut Formatter) -> fmt::Result {
         // As a UX optimization, display "-1" for the largest field element.
+        //TODO: INTERNAL: check to see if there would be any parsing of the -1 value that could error
```

Comment:
This may be unnecessary.

Reply:
Now I'm a bit confused about this. The code is `self.is_one()` which means that we should display `1` so `-1` doesn't make sense here. Please double-check that this is the wrong situation for a negative here.

The other case, if we're looking for `F::order() - 1` would make sense, as long as our diagnostic logging has the field order already known, which is the reason that I wanted to note this temporarily. Can we be sure that it'll be in the logs so there's no ambiguity?

Reply:
Whoops -- yeah looks like a mistake. We could check for equality with `largest_element()`, or just remove the case. Good point about the ambiguity.